express-rest-api/server/routes/users.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with a resource');
```

Comment:
It also would be better with `res.json`.

Reply:
I will change this response to show the list of users!